packages/api/src/routes/medical/fhir-r4-proxy.ts

```diff
@@ -17,6 +35,53 @@ const fhirRouter = (fhirServerUrl: string) =>
       log(`Proxying to FHIR server: ${updatedURL}`);
       return updatedURL;
     },
+    //eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do we need this eslint? Can't we just get rid of whatever the unused variable is here?

Reply:
Fixed.

I had done it originally as it's part of the boilerplate code for this function and I thought it might helpful to know the `userReq` is available. But if you hover over `userResDecorator` you can see that anyway.